### PR TITLE
Remove all DataUnion code from client

### PR DESCRIPTION
**Pull request by timoxley**
Removed all DataUnion code from client.

New standalone DU client will be developed under: https://github.com/dataunions/data-unions

## Checklist

- [x] Has passing tests that demonstrate this change works
- [x] Updated Changelog
- [x] Updated Documentation

### CODE REVIEW

File: packages/client/src/config.schema.json [hunk 1]
```diff
@@ -1,384 +1,333 @@
 {
-    "$id": "config.schema.json",
```
Comment (teogeb) [inline packages/client/src/config.schema.json]: Could keep the original 4 space indentation

File: packages/client/test/end-to-end/dataunion/LoginEndpoints.test.ts [hunk 2]
```diff
@@ -1,89 +0,0 @@
-import assert from 'assert'
```
Comment (teogeb) [inline packages/client/test/end-to-end/dataunion/LoginEndpoints.test.ts:1]: Remove also `LoginEndpoints.ts` and `Session.ts` as those are not needed anymore

File: packages/client/src/Config.ts [hunk 3]
```diff
@@ -93,25 +79,23 @@ export type StrictStreamrClientConfig = {
...
     /** @internal */
     _timeouts: TimeoutsConfig
     /** @internal */
     debug: DebugConfig
 } & (
-    EthereumConfig
-    & ConnectionConfig
-    & SubscribeConfig
-    & EncryptionConfig
-)
```
Comment (teogeb) [inline packages/client/src/Config.ts:111]: Keep the original indentation?

File: packages/client/test/end-to-end/dataunion/LoginEndpoints.test.ts (hunk 2 above)
Comment (timoxley) [reply packages/client/test/end-to-end/dataunion/LoginEndpoints.test.ts:1]: Done

File: packages/client/src/Config.ts (hunk 3 above)
Comment (timoxley) [reply packages/client/src/Config.ts:111]: Done

File: packages/client/src/config.schema.json (hunk 1 above)
Comment (timoxley) [reply packages/client/src/config.schema.json]: Done

**Review by harbu — APPROVED** [review]
LGTM!

**Comment by harbu** [comment]
feel free to land @timoxley

**Pull request merged**